marklogic-data-hub-central/ui/e2e/qa-project/gradle-dhs.properties

```diff
@@ -0,0 +1,45 @@
+mlDHFVersion=5.3-SNAPSHOT
+
+mlIsHostLoadBalancer=true
+mlIsProvisionedEnvironment=true
+mlStagingAuth=basic
+mlFinalAuth=basic
+mlJobAuth=basic
+mlAppServicesAuthentication=basic
+mlJobSimpleSsl=true
+mlFinalSimpleSsl=true
+mlStagingSimpleSsl=true
+mlAdminScheme=https
+mlManageScheme=https
+mlAppServicesSimpleSsl=true
+mlManageSimpleSsl=true
+sslFlag=true
+
+mlStagingAppserverName=data-hub-STAGING
```

Comment:
Can remove all of these mlStaging* properties, assuming they're already defined in gradle.properties